src/store/index.ts

```diff
@@ -86,7 +87,7 @@ export type AppReduxState = CombinedState<{
   [CURRENCY_VALUES_KEY]: CurrencyValuesState
   [COOKIES_REDUCER_ID]: Map<string, any>
   [ADDRESS_BOOK_REDUCER_ID]: AddressBookReducerMap
-  [CURRENT_SESSION_REDUCER_ID]: Map<string, any>
+  [CURRENT_SESSION_REDUCER_ID]: SerializedSessionState
```

Comment:
Previously we used to prefix wit `Serialized` locally stored values of states which actual value is Immutable.js instances because we'd convert it to plain JSON since it doesn't have sense to store the immutable.js instances in localstorage

Reply:
Changed the naming convention